scss/_utilities.scss

```diff
@@ -0,0 +1,50 @@
+// Styling Utilities
+
+.n-content-u-margin--left-right {
+	margin-left: 16px;
+	margin-right: 16px;
+}
+
+.n-content-u-margin--bottom-none {
+	margin-bottom: 0;
+}
+
+.n-content-u-padding--left-right {
+	padding-left: 16px;
+	padding-right: 16px;
+}
+
+.n-content-u-padding--bottom {
+	padding-bottom: 16px;
+}
+
+.n-content-u-padding--bottom-none {
+	padding-bottom: 0;
+}
+
+.n-content-u-background-color--pink {
+	background-color: getColor('pink');
+}
+
+.n-content-u-background-color--page {
+	background-color: getColorFor('page', 'background');
+}
+
+.n-content-u-border--all {
+	border: 1px solid getColor('warm-3');
+}
+
+.n-content-u-border--left {
+	border-left: 1px solid getColor('warm-3');
+}
+
+.n-content-u-align-center {
+	text-align: center;
+}
+
+//Want this to be capital L for consistency with o-grid definition
+.n-content-u-hide--L { 	// scss-lint:disable SelectorFormat
```

Comment:
We should provide hide/show utils for all breakpoints or none at all. I would probably go for the latter so it's entirely up to the implementation.